app/models/gem_download.rb

```diff
@@ -1,4 +1,6 @@
 class GemDownload < ApplicationRecord
+  include SemanticLogger::Loggable
```

Comment:
can we make ApplicationRecord's default logger be the semantic one, so we don't have to one-off include the module in individual record classes?